### DISCUSSION TITLE
Groq API instead of OpenAI for Pile?

### DISCUSSION THREAD

**Discussion by morganschp** [UdaraJay/Pile#108] [Q&A]
I would love to use the Pile diary app, but using Groq's API. From what I understand Groq is designed to be OpenAI-compatible in structure - but I can't make it work. Could someone clarify if it's even possible & what Base URL and model name one should enter to make it work?

**Comment by UdaraJay** [comment]
Replace the base URL in the openai setting tab with the grok-endpoint, model and API key - it's compatible with any API that is oai compatible.
